docs/chapters/advanced/computed-fields.md

```diff
@@ -3,7 +3,14 @@
 A **computed field** is a field in an entity that is not in the data source, but is the result of processing using the value of other fields. A computed field is then ignored in each insert or edit operation and is automatically populated by the system in each read operation where site is requested.
 
 Typetta offers convenient middleware that can be used to define calculated fields. The following example shows how you can define a ``fullName`` field that is the composition of ``firstName`` and ``lastName`` of a hypothetical entity that represents a user. These last two fields are actually historicised in the data source, while ``fullName`` is in fact a virtual field.
-
+```graphql
```

Comment:
github pages markdown does not support graphql language, please use typescript as a default